### ISSUE TITLE
test_set accuracy

### ISSUE THREAD

**Issue by 853498747** [xuehy/Cross-View-Gait-Deep-CNNs#4]
Excuse me, I ran your code. The validation set works well, but why is the result of the test set much worse than the DeepCNNs paper?

**Comment by xuehy** [comment]
How many iterations did you run? Did you select the parameters when the validation accuracy is the highest?

**Comment by 853498747** [comment]
iteration = 300K and validation_set accuracy of 0.96.
The test_set results are only a little better than the form you posted, and there is a big gap between the results in the DCNNS paper.

**Comment by xuehy** [comment]
My form is not from the final result. In your training procedure, is the validation accuracy still increasing? You may try to run more iterations. Someone has offered me feedbacks that the training takes a lot of iterations to converge.

**Comment by 853498747** [comment]
After iterations morethan  300K, the accuracy of the verification set can no longer be increased. I still have some confusion about them. I will ask you after I have a deep understanding of the code and the paper.Thank you for your patience.
